observation/night sky/lores.md

```diff
@@ -6,8 +6,43 @@ grand_parent: Observation
 nav_order: 1
 ---
 
-## Mythical Lores
+## Greek Lores
 
-### Greek
+As you look up at the night sky, Uranus, the primordial Greek sky god, appears alongside Jupiter, the Roman King of the Gods, with Orion, the hunter, and Cetus, the sea monster. According to Greek mythology, Perseus slayed Cetus, a sea monster, to save Andromeda, the beautiful daughter of Cassiopeia and Cepheus, who decided to sacrifice her to the gods after an oracle made the suggestion. Astronomy is filled with fascinating myths and lores that have played a huge role in shaping its history giving it its current form.
 
-### Vedic
+The Greeks imagined the heavens as a great, solid dome, which, some say, was forged of bronze, and upon which the heavenly constellations were fixed. The Titan Atlas, who stood either beneath the axis of heaven in the far north, was said to spin the dome around upon his shoulders, causing the stars to rise and set.
+Part of the heavenly dome always lay beneath the horizon. Here the constellations were apparently believed to dwell deep beneath the earth in the misty pit of Tartaros, or else within the lands of the dead. When they rose up into the heavens, the constellations were first bathed in the purifying waters of the great earth-encircling river Okeano. Various myths describe the birth and death of the semi-immortal constellations: such as the Gemini twins, or Dioskouroi, who were said to divide their time equally between Heaven and Hades. Orion was also described by Homer as both striding across the heavens and hunting wild beasts in the underworld.
+
+
+### Scorpio and Orion
+
+Scorpius was a giant scorpion sent by Gaia, the Earth to slay the giant Orion when he threatened to slay all the beasts of the world. Orion and the Scorpion were afterwards placed amongst the stars as the constellations of the same name. The two opponents are never seen in the sky at the same time, for as one constellation rises, the other sets. The ancient Greek Skorpios originally encompassed two constellations, Scorpio formed its body and Libra its claws.
+
+
```

Comment:
You can insert a fun fact here that the two constellations are nearly opposite each other in the night sky.

Following the arrow in the bow of Orion leads towards Scorpio, and vice versa, following the tail of scorpio leads towards Orion.

Of course, phrased better.

Rest seems fine